flowapi/app/geography.py

```diff
@@ -0,0 +1,81 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from quart import Blueprint, current_app, request, stream_with_context, jsonify
+from .check_claims import check_claims
+
+blueprint = Blueprint(__name__, __name__)
```

Comment:
I suggest to pass an explicit name for the first argument here:
```suggestion
blueprint = Blueprint("geography", __name__)
```

We should also do this for the blueprint in `run_query.py` and tweak the call to `url_for(...)` accordingly.